packages/react-ui/src/theme/sections/components/dataDisplay.js

```diff
@@ -539,15 +536,15 @@ export const dataDisplayOverrides = {
   // Svg Icons
   MuiSvgIcon: {
     styleOverrides: {
-      root: {
+      root: () => ({
         fontSize: ICON_SIZE_MEDIUM
-      },
-      fontSizeSmall: {
+      }),
+      fontSizeSmall: () => ({
         fontSize: ICON_SIZE_SMALL
-      },
-      fontSizeLarge: {
+      }),
+      fontSizeLarge: () => ({
         fontSize: ICON_SIZE_LARGE
-      }
+      })
```

Comment:
Do we need function here? even if value is a const?

Reply:
No, not really, but I wanted to be very explicit to avoid mistakes in the future... We can safely remove those if you want.